### PR TITLE
Minor README tweaks and typo fixes

### PR DESCRIPTION
**Pull request by nchammas**
Happy to tweak further (or revert edits) as necessary.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -56,8 +58,6 @@ Links
 
 * Feedback and ideas: http://mail.python.org/mailman/listinfo/code-quality
 
-* Cyclomatic complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity.
```
Comment (sigmavirus24) [inline README.rst]: This belongs here and should stay.

Comment (nchammas) [inline README.rst]: Restored! (Minus the trailing period.)

**Comment by sigmavirus24** [comment]
Thanks @nchammas!

File: README.rst [hunk 2]
```diff
@@ -56,11 +56,13 @@ Links
...
+* Cyclomatic complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity
 
 * Ned Batchelder's script:
   http://nedbatchelder.com/blog/200803/python_code_complexity_microtool.html
 
+* McCabe complexity: http://en.wikipedia.org/wiki/Cyclomatic_complexity
```
Comment (nchammas) [inline README.rst]: Hmm, I'm not sure how this ended up here. My intention was to make the first mention of "McCabe complexity" at the start of the README a link.

@sigmavirus24 - I can remove this duplicate link from here and put it on that first mention if you want.

Comment (sigmavirus24) [inline README.rst]: That ended up there, because I updated your branch to match the rest of the README. This is exactly where I want it. =)

**Pull request merged**